### ISSUE TITLE
Data are wrong (accuweather instead of openweather)

### ISSUE THREAD

**Issue by shumafuk** [WangDaYeeeeee/GeometricWeather#402]
In my hometown, data from openweahter are more accurate than from accuweahter. However, geometric weather shows mi data from accuweather, no matter which weather source is selected. Should I set something else except "weather source" in settings to get different data?

**Comment by papjul** [comment]
Yes, you need to add a new location from the OpenWeather provider in the search, so you can have twice the same location from different providers.

You also need your own API key in the advanced settings, if not already done.

**Comment by shumafuk** [comment]
Thanks. Do we have some guide for this? Should API be with endpoint?

**Comment by papjul** [comment]
You only need to do the search like you did for Accu Weather.

For API, you only need an API key, see FAQ "How to get an API key" https://openweathermap.org/faq

**Comment by shumafuk** [comment]
I did it as written but data cannot be loaded. "Request weather data failed."

**Comment by shumafuk** [comment]
I have probably same issue as mentioned here: https://github.com/WangDaYeeeeee/GeometricWeather/issues/395

My API url with my hometown has data that can be seen in browser just fine.

**Comment by sheltpat** [comment]
> I have probably same issue as mentioned here: #395
> 
> My API url with my hometown has data that can be seen in browser just fine.

Same for me, request failing and code 400 ("nothing to geocode") via browser API call.
 
Is this an OpenWeatherMap issue? I get the result on their website search just fine.

Sorry to post in both issues, not sure if both get the same visibility.

Edit: This link returns values just fine:
https://api.openweathermap.org/data/2.5/weather?q=[City,2LetterContryCode]&appid=[APIKey]

Maybe a wrong syntax for OpenWeatherMap queries?

Edit2: Somehow switching between providers had my app show failed requests regardless of provider. App data deletion fixed it (default accuweather settings).

Edit3: Downgraded to 3.100_fdroid, OpenWeatherMap works fine again. Possibly a bug in 3.102.
